pages/packages/[package]/versions/[version]/topics/[topic].tsx

```diff
@@ -132,11 +132,7 @@ export default function TopicPage({ topicData }: Props) {
                 return (
                   <div key={section.name}>
                     <h2>{section.name}</h2>
-                    {section.name?.toLowerCase()?.includes('argument') ? (
-                      <Html className="arguments">{section.description}</Html>
-                    ) : (
-                      <Html>{section.description}</Html>
-                    )}
+                    <Html className="arguments">{section.description}</Html>
```

Comment:
You should keep this logic, there is a reason we added it :)
The classname changes the layout

Reply:
But its worth adding a comment to explain actually, you can add `// This is needed for legacy code where Arguments moved to the section array and need some extra styling to be rendered differently from other sections`

Reply:
Oh ok if the Format section also needs this, then add a condition to include that class as well for that section